float64 xRobot
float64 yRobot
float64 thetaRobot
string color
---
int64 figure1
float64 confidence1
int64 figure2
float64 confidence2